localisation/customizable_localisation/wc_customizable_localisation.txt

```diff
@@ -1306,19 +1306,25 @@ defined_text = {
 		}
 		localisation_key = GetClassSkill_the_dagger
 	}
+	# text = {
+		# trigger = {
+			# is_hunter_class_trigger = yes
+			# prefers_guns_trigger = yes
+		# }
+		# localisation_key = GetClassSkill_the_gun
+	# }
+	# text = {
+		# trigger = {
+			# is_hunter_class_trigger = yes
+			# prefers_guns_trigger = no
```

Comment:
But what about the trigger? It's not used anywhere except this place.

Reply:
I want to use that trigger later